tests/slow/src/test/scala/tests/scalacli/ScalaCliSuite.scala

```diff
@@ -227,6 +228,18 @@ class ScalaCliSuite extends BaseScalaCliSuite(V.scala3) {
       _ <- server.initialized()
       _ = FileLayout.fromString(simpleFileLayout, workspace)
       _ = FileLayout.fromString(bspLayout, workspace)
+      _ <- server.fullServer
+        .didChangeWatchedFiles(
```

Comment:
Is that needed? Shouldn't the file watcher detect the change?

Reply:
File watcher detects changes from the other `didChangeWatchedFiles`, but this change is from lsp `workspace/didChangeWatchedFiles`. I'm not sure why we need both and which notifications should belong where

Reply:
My understanding is that we need both, since not all editors do `workspace/didChangeWatchedFiles` (at least not reliably), so `workspace/didChangeWatchedFiles` is sort of "additional". @tgodzik, right?

Reply:
That's right, at least from what I remember. For build tool files we use the LSP since this should be fairly reliable.